fiftyone/core/annotation/validate_label_schema.py

```
@@ -0,0 +1,585 @@
+        )
+
+    if len(value) > foac.VALUES_THRESHOLD:
+        raise ValueError(
+            f"'values' setting for field '{field_name}' has more than "
+            f"{foac.VALUES_THRESHOLD} values"
+        )
+
+    if len(value) > len(set(value)):
+        raise ValueError(
+            f"'valaues' setting for field '{field_name}' has duplicates"
+        )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)